packages/protobuf/src/from-json.ts

```diff
@@ -702,7 +690,7 @@ function timestampFromJson(timestamp: Timestamp, json: JsonValue) {
     ms > Date.parse("9999-12-31T23:59:59Z")
   ) {
     throw new Error(
-      `cannot decode message ${timestamp.$typeName} from JSON: must be from 0001-01-01T00:00:00Z to 9999-12-31T23:59:59Z inclusive`,
+      `cannot decode ${timestamp.$typeName} from JSON: must be from 0001-01-01T00:00:00Z to 9999-12-31T23:59:59Z inclusive`,
```

Comment:
We're omitting `message` in all other places for the wkt. Changing it here for consistency.